src/app/lib/config/services/news.js

```diff
@@ -53,7 +53,6 @@ export const service = {
     isTrustProjectParticipant: true,
     script: cyrillicAndLatin,
     manifestPath: '/articles/manifest.json',
-    swPath: '/articles/sw.js',
```

Comment:
Are you missing sport and newsround?

Reply:
Sport & Newsround didn't have the `swPath` specified in the first place.